src/Radicle/Internal/Core.hs

```diff
@@ -629,6 +624,10 @@ specialForms = Map.fromList $ first Ident <$>
               _ -> throwErrorHere $ SpecialForm "fn" "First argument must be a vector of argument atoms"
           _ -> throwErrorHere $ SpecialForm "fn" "Need an argument vector and a body"
       )
+  , ("scope", \case
```

Comment:
This doesn't seem to be used anywhere? And isn't it essentially a self-calling lambda?